### PR TITLE
dev-python/rebulk: verbump to 1.0.0

### PR DESCRIPTION
**Pull request by Coacher**
Signed-off-by: Ilya Tumaykin <itumaykin@gmail.com>
Package-Manager: Portage-2.3.51, Repoman-2.3.11

### CODE REVIEW

File: dev-python/rebulk/rebulk-1.0.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```
Comment (Zlogene) [inline dev-python/rebulk/rebulk-1.0.0.ebuild]: why not 7?

File: dev-python/rebulk/rebulk-1.0.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+	rm runtests.py || die
+
+	# Disable unconditional dependency on dev-python/pytest-runner.
+	sed -i -e "s|'pytest-runner'||g" setup.py || die
+
+	distutils-r1_python_prepare_all
+}
+
+python_test() {
+	esetup.py test
```
Comment (mgorny) [inline dev-python/rebulk/rebulk-1.0.0.ebuild]: Maybe you'd be able to kill pytest-runner altogether and call pytest directly?

File: dev-python/rebulk/rebulk-1.0.0.ebuild [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+inherit distutils-r1
+
+DESCRIPTION="Python library that performs advanced searches in strings"
+HOMEPAGE="https://github.com/Toilal/rebulk https://pypi.org/project/rebulk/"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="MIT Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="test"
```
[the remark this answers is not in the document]
Comment (Coacher) [reply dev-python/rebulk/rebulk-1.0.0.ebuild]: I remember reading a discussion on g-dev ML about this and the general consensus was not to add this line into hundreds of ebuilds in tree. Please make a reference to a current policy or guideline for this change.

File: dev-python/rebulk/rebulk-1.0.0.ebuild (hunk 2 above)
Comment (Coacher) [reply dev-python/rebulk/rebulk-1.0.0.ebuild]: I'll look into it, yes.

File: dev-python/rebulk/rebulk-1.0.0.ebuild (hunk 3 above)
[the remark this answers is not in the document]
Comment (mgorny) [reply dev-python/rebulk/rebulk-1.0.0.ebuild]: 'General consensus' or the few loud complainers? There's no policy to the contrary, so PMS holds. If the package fails to build with `FEATURES=test USE=-test`, it's broken and you need to fix it.

**Comment by mgorny** [comment]
Ping.

**Comment by mgorny** [comment]
Closing due to no activity. Feel free to reopen once you are available again.

**Pull request closed without merging**